src/openforms/config/migrations/0031_auto_20220718_1544.py

```diff
@@ -0,0 +1,61 @@
+# Generated by Django 3.2.14 on 2022-07-18 13:44
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("config", "0030_piwik_pro_support"),
+    ]
+
+    operations = [
+        migrations.DeleteModel(
+            name="CSPSetting",
+        ),
+        migrations.AddField(
```

Comment:
I think these migration operations can be cleaned up and/or re-generated while moving the model to the new app